README.md

```diff
@@ -4,9 +4,23 @@ This repository implements a simple cron job that makes sure all Arcadia Science
 
 ## Motivation
 
-<!-- TODO: Add more details -->
+When a user within the Arcadia Science organization, that repository does not have any branch protections. This means the user can push/force push to any branch. This is suboptimal because we'd like most major changes to go through code review. GitHub conveniently offers easily customizable branch protection rules, but these can only be set at the repository level and no organization-wide defaults exist.
```

Comment:
```suggestion
When a user within the Arcadia Science organization creates a new Github repository, that repository does not have any branch protections. This means the user can push/force push to any branch. This is suboptimal because we'd like most major changes to go through code review. GitHub conveniently offers easily customizable branch protection rules, but these can only be set at the repository level and no organization-wide defaults exist.
```